src/constants.js

```diff
@@ -38,6 +37,7 @@ consts.ENV_VARS = {
     INTERNAL_PORT: 'APIFY_INTERNAL_PORT',
     MEMORY_MBYTES: 'APIFY_MEMORY_MBYTES',
     LOG_LEVEL: 'APIFY_LOG_LEVEL',
+    ACTOR_EVENTS_WEB_SOCKET: 'APIFY_ACTOR_EVENTS_WEB_SOCKET',
```

Comment:
I think this name is not great, it's not a web socket, but a web socket URL, right?
 Maybe `APIFY_ACTOR_EVENTS_WS_URL` ?